_includes/operator/install.md

```diff
@@ -0,0 +1,71 @@
+## Install StorageOS operator
+
+The StorageOS cluster operator is a [Kubernetes native application](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/) developed to
+deploy and configure StorageOS clusters, and assist with maintenance
+operations.
+
+The StorageOS operator can be installed with Helm.
+
+### Install
+
+```bash
+helm repo add storageos https://charts.storageos.com
+helm install storageos/storageoscluster-operator --namespace storageos-operator
+```
+
+> The Helm chart can be found in the [Charts public
+> repository](https://github.com/storageos/charts).
+
+The operator is a Pod manged by a Deployment Controller. Once the Pod is ready,
+a StorageOS cluster definition can be created. The operator will deploy a
+StorageOS cluster based on the configuration specified in the cluster
+definition. 
+
+### Create a Secret
+
+Before deploying a StorageOS cluster, create a Secret to define the StorageOS
+api Username and Password in base64 encoding.
+
+```bash
+kubectl create -f - <<END
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "storageos-api"
+  namespace: "default"
+  labels:
+    app: "storageos"
+type: "kubernetes.io/storageos"
+data:
+  # echo -n '<secret>' | base64
+  apiUsername: c3RvcmFnZW9z
+  apiPassword: c3RvcmFnZW9z
+END
+```
+
+## Trigger a StorageOS installation
+
+> This is a Cluster Definition example. Checkout the `spec` parameters
+> available in the [Resource Configuration
+> page](https://github.com/storageos/cluster-operator#storageoscluster-resource-configuration).
+
+```bash
+kubectl create -f - <<END
+apiVersion: "storageos.com/v1alpha1"
+kind: "StorageOSCluster"
+metadata:
+  name: "example-storageos"
+    namespace: "default"
+    spec:
+      secretRefName: "storageos-api" # Reference from the Secret created in the previous step
+      secretRefNamespace: "default"  # Namespace of the Secret
+      images:
+        nodeContainer: "storageos/node:1.0.0-rc5" # StorageOS version
+      resources:
+        requests:
+          memory: "128Mi"
+END
+```
+
+You can find more examples such as deploy with CSI or deploy a non embedded kv
```

Comment:
Maybe `a non embedded` -> `an external`?

Reply:
I struggled with that as external is not the right word for me either. It is external of StorageOS, but external of Kubernetes?
One of the examples I have for that is etcd as a k8s svc using the etcd operator and another example using a etcd cluster outside of the k8s and referencing using an external SVC. So I'm not sure about the wording. 
Any other suggestion or thought? Do you think we should put `external` anyway?

Reply:
I think external is good way to phrase it but perhaps we need to qualify its use. If you were planning on linking to another docs page that shows a StorageOS cluster with external etcd then I suppose that'd be enough explanation.

Reply:
A applied a change. What do you think?

Reply:
done